app/Http/Controllers/User/BookController.php

```diff
@@ -84,11 +81,8 @@ public function getBookCategory($slug)
     {
         $id = last(explode('-', $slug));
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Formatting.DisallowMultipleStatements.SameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>**Line**: 85</br>Each PHP statement must be on a line by itself